libp2p.nimble

```diff
@@ -137,7 +92,6 @@ task pin, "Create a lockfile":
   exec nimc & " c -r tools/pinner.nim"
 
 import sequtils
```

Comment:
nit: not sure about this import here. i think we should keep it on the top of file (maybe below `requires` block).